### DISCUSSION TITLE
Merge  on other user git.

### DISCUSSION THREAD

**Discussion by 2dof** [github/codeql#12526] [General]
I just find out that someone ( user @redsun82) made pull request all codeql project ( forked from github/codeql) on My 
repo account at 7 march 2023 (https://github.com/2dof/codeql) - is this some github error or some security vulnerability to  load 
someone's projects on  other user git (without knowing password) who is unrelated to the project ? 

I want to delete this project from my repo so please @redsun82  let me know if You have backup.

**Comment by d10c** [comment]
Hi @2dof!

It sounds like all is in order, as the merge commits from @redsun82 and others originally appeared on the upstream (github/codeql) main branch, which was then pulled into the main branch of your fork (2dof/codeql) on March 7. There is no security issue, because the PR in question was merged into upstream, and not your fork; otherwise you would have seen a PR pop up in the Pull Requests tab of your fork before it got merged.

So feel free to delete this repo from your account.

**Reply by 2dof** [reply]
thanks for fast answer
to clarify: I do not created fork (2dof/codeql)  (not I was logged  that day but not even performing any pulls on my own repos (only commits of md. docs ) and did not forked others projects , that I found surprising because, I'm not even correlated/interested in codeql.
